glue-codes/openfast/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 #
 
-add_executable(openfast
-  src/FAST_Prog.f90)
+add_executable(openfast src/FAST_Prog.f90)
+set_source_files_properties(src/FAST_Prog.f90 PROPERTIES COMPILE_FLAGS "-fno-var-tracking -fno-var-tracking-assignments")
```

Comment:
heh... I was just griping about this on eagle. Great addition!

Reply:
Yes, I've also added this to both of those modules.

Reply:
@sayerhs do you have any input on whether this should always be disabled or if only for Release mode (`-O3`)? I think I see this warning with `-O2` as well, but RelWithDebInfo includes `-g` so I think we may want variable tracking there.

Reply:
@rafmudaf My vote would be to **not** add `-fno-var-tracking...` flags for `Debug` build, but use it for `Release` and `RelWithDebInfo`. With `-O2 -g` a lot of variables are removed anyway, so I am not sure it is going to affect much.